### DISCUSSION TITLE
Can't compile with esp32 core 3.0.0-alpha2

### DISCUSSION THREAD

**Discussion by whogarden** [schreibfaul1/ESP32-audioI2S#615] [Q&A]
Hello,
I can't compile the sketch in [README.md].
Using : 
Arduino IDE 1.8.19
esp32 core 3.0.0-alpha2
ESP32 Dev Module
Library from here : https://github.com/schreibfaul1/ESP32-audioI2S
error : ESP32-audioI2S-master\src\Audio.cpp:287:33: error: 'PERIPHS_IO_MUX_GPIO0_U' was not declared in this scope
          ESP32-audioI2S-master\src\Audio.cpp:287:57: error: 'FUNC_GPIO0_CLK_OUT1' was not declared in this scope
etc.
Does it works with esp32 core 3.0.0-alpha2?
Thank you

**Comment by schreibfaul1** [comment]
Hello,
the "3.0 alpha" versions are not yet completely stable. With Arduino V3.0 there is the IDF V5.1 which changes some things in the I2S driver. 
There is a branch "Arduino 3.0"
 ![image](https://github.com/schreibfaul1/ESP32-audioI2S/assets/26044260/a0976da1-9e3c-41b1-83c2-417eb6719699) 
but I can't guarantee that it will work with the Arduini IDE. We are waiting for a stable release of Espressif.

**Comment by whogarden** [comment]
Thank you, 
you are right a stable version is the right solution
